src/embezzler.ts

```diff
@@ -316,13 +316,18 @@ export function embezzlerCount(): number {
 }
 
 export function estimatedTurns(): number {
+  // Assume roughly 2 fullness from pantsgiving and 8 adventures/fullness.
+  const pantsgivingAdventures = have($item`Pantsgiving`)
+    ? Math.max(0, 2 - get("_pantsgivingFullness")) * 8
+    : 0;
+  const nightcapAdventures = globalOptions.ascending && myInebriety() <= inebrietyLimit() ? 60 : 0;
+  const thumbRingMultiplier = have($item`mafia thumb ring`) ? 1.04 : 1;
```

Comment:
I feel the need to say it's actually 1/0.96 but I think we can call that negligible

Reply:
yep